configcatclient/configcatclient.py

```diff
@@ -37,8 +38,10 @@ def get(cls, sdk_key, options=None):
             client = cls._instances.get(sdk_key)
             if client is not None:
                 if options is not None:
-                    client.log.warning('Client for sdk_key `%s` is already created and will be reused; '
-                                       'options passed are being ignored.' % sdk_key)
+                    client.log.warning('There is an existing client instance for the specified SDK Key. ' \
+                                       'No new client instance will be created and the specified options are ignored. ' \
```

Comment:
remove the unnecessary blackslashes

```suggestion
                    client.log.warning('There is an existing client instance for the specified SDK Key. '
                                       'No new client instance will be created and the specified options are ignored. '
```